<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewEmployee</class>
 <widget class="QDialog" name="AddNewEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增员工</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	border-style:outset;
	background-color:rgba(190, 253, 190);
	border-width:4px;
	border-radius:10px;
	border-color:rgba(200,200,255,255);
	font:bold 16px;
	color:rgba(0,0,0,255);
	padding:4px;
}
QPushButton:pressed{
    border-style:inset;
}
QPushButton:hover{
    background-color:rgba(148, 221, 132);
    color:rgba(0,0,0,200);
}
QDialog
{background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>员工编号：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="E_ID">
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>登陆密码：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="E_PASSWORD">
     <property name="maxLength">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>员工姓名：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="E_NAME">
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>员工性别</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="E_SEX">
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>电话号码</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="E_PHONE">
     <property name="maxLength">
      <number>11</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>入职日期</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDateTimeEdit" name="E_ENTRYDATE">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>员工类别</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QComboBox" name="E_TYPE">
     <item>
      <property name="text">
       <string>销售</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>维修</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="Line" name="line_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>72</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="1">
    <widget class="QPushButton" name="confirm">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
